graphql_compiler/compiler/expressions.py

```diff
@@ -394,10 +394,19 @@ def to_cypher(self):
                              u'should not be called.')
 
     def to_sql(self, aliases, current_alias):
-        """Not implemented, should not be used."""
-        raise AssertionError(u'GlobalContextField is not used as part of the query emission '
-                             u'process in SQL, so this is a bug. This function '
-                             u'should not be called.')
+        """Return a sqlalchemy Column picked from the appropriate alias."""
+        self.validate()
+        if isinstance(self.field_type, GraphQLList):
+            raise NotImplementedError(u'The SQL backend does not support lists. Cannot '
+                                      u'process field {}.'.format(self.location.field))
+
+        if self.location.field is not None:
+            if '@' in self.location.field:
+                raise NotImplementedError(u'The SQL backend does not support __typename.')
+            return aliases[(self.location.at_vertex().query_path, None)].c[self.location.field]
+        else:
+            raise AssertionError(u'This is a bug. The SQL backend does not use '
+                                 u'global context fields to point to vertices.')
```

Comment:
Print some more context. If we ever see this message, all we'll know is that we have a bug -- we won't know any contextual info or even what field caused it.